### DISCUSSION TITLE
(413) Request Entity Too Large

### DISCUSSION THREAD

**Discussion by robin-mi89** [weather-gov/api#351] [Q&A]
I have a program that is trying to download hourly forecast information from the /gridpoints/{wfo}/{x},{y}/forecast/hourly uri. However, every request made 3 hours after the initial request gets back a (413) Request Entity Too Large error. Do you know why I might be receiving this error?

**Comment by StephenClouse** [comment]
That's certainly a new one.

I assume you're not actually sending a request entity (which wouldn't make much sense, since API is all GET requests).

Can you capture the full request and response (including headers) when this occurs? Otherwise there's not a whole lot I can tell you. 4xx errors imply something is wrong with the client, not the server.

**Comment by scadergit** [comment]
Closing from no-reply.
